pmd-apex/src/main/java/net/sourceforge/pmd/lang/apex/ast/ASTUserEnum.java

```diff
@@ -22,8 +23,10 @@ public Object jjtAccept(ApexParserVisitor visitor, Object data) {
 
     @Override
     public String getImage() {
-        String apexName = getDefiningType();
-        return apexName.substring(apexName.lastIndexOf('.') + 1);
+//        String apexName = getDefiningType();
+//        return apexName.substring(apexName.lastIndexOf('.') + 1);
+        // TODO(b/239648780)
+        return "";
```

Comment:
Just curious: why didn't the existing code work?

Reply:
Currently there's no implementation for `getDefiningType` in [`AbstractApexNode`](https://github.com/eklimo/pmd/blob/summit/pmd-apex/src/main/java/net/sourceforge/pmd/lang/apex/ast/AbstractApexNode.java#L94), resulting in a NPE when `getImage` is called.